### ISSUE TITLE
Incorrect block height in snapshot name

### ISSUE THREAD

**Issue by sheremetyev** [oxheadalpha/xtz-shots-website#6]
https://mainnet.xtz-shots.io/ shows "Block height: 2260016" (and it's included in snapshot file name), but links to TzStats and TzKT (for `BLAWdQTsauNC6y13MsnLQQuhPabsno86t1XJrL4fQ2CJRnBFdL9 `) show block 2,260,014.

**Comment by orcutt989** [comment]
Thanks for catching this. 

This appears to only be the case that rolling artifacts are displaying 2 blocks ahead on the web page, but are actually 2 blocks behind.  This is only a web-page issue. This was a part of our block finalization logic implemented at the suggestion of the Tezos devs.

The block displayed on TzStats and TzKT is correct as this is the hash of the head block of the artifact we have provided.

We have fixed the issue and it will reflect on the page in the next few hours when the next batch of artifacts generate.
